templates/terraform/resource.erb

```diff
@@ -478,7 +478,9 @@ func resource<%= resource_name -%>Import(d *schema.ResourceData, meta interface{
 <%= lines(compile(object.custom_code.custom_import)) -%>
 <% else -%>
     config := meta.(*Config)
-    parseImportId([]string{"<%= import_id_formats(object).map{|s| format2regex s}.join('","') -%>"}, d, config)
+    if err := parseImportId([]string{"<%= import_id_formats(object).map{|s| format2regex s}.join('","') -%>"}, d, config); err != nil {
```

Comment:
Do you mind splitting this to a separate PR? I can see the case for the generated test stuff but this is a stretch to include.

Reply:
This makes sense! But I think it's a change that would also make sense to apply outside of this PR, and by doing that we'll create a nicer diff in that PR + in this PR. If we spin it out, submit it, and then rebase off HEAD again we'll have submitted 2 separate, easier to understand changes.

Reply:
Pulled out into https://github.com/GoogleCloudPlatform/magic-modules/pull/1091 - I'll rebase this onto master once that goes in.